migrations/frontend/1663665519_perforce_merge_includes_excludes_columns/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE ONLY sub_repo_permissions
+    ADD COLUMN IF NOT EXISTS paths text[];
+
+COMMENT ON COLUMN sub_repo_permissions.paths IS 'Paths that begin with a - are exclusion paths.';
```

Comment:
Probably this will read better
```suggestion
COMMENT ON COLUMN sub_repo_permissions.paths IS 'Paths that begin with a minus sign (-) are exclusion paths.';
```

Reply:
Thanks! Yeah that's much better